### DISCUSSION TITLE
Alo t tạo project tuần một rồi :v

### DISCUSSION THREAD

**Discussion by thaopham2501** [baonhu1304/Lop-10A2-nhom-1#31] [General]
Giờ vô phân công với bắt đầu tìm ca dao tục ngữ đi, có gì hay hay cứ ném vô đó , xong ghi mấy dòng cảm nghĩ dzo

**Comment by thaopham2501** [comment]
chịu luông :)) @baonhu1304  @letu25082006  @NguyenNgocNhuY  @Lenga10A2

**Comment by Lenga10A2** [comment]
Bình tễnh để t kiếm mấy câu ca dao cho

Vào 11:28, T.3, 22 Th2, 2022 Phạm Ngọc Hương Thảo ***@***.***>
đã viết:

**Comment by Lenga10A2** [comment]
Ủa mà cao dao về chủ đề j

Vào 11:52, T.3, 22 Th2, 2022 Nga Lê ***@***.***> đã viết:

**Reply by HienCaoPhuNhuan** [reply]
> Ủa mà cao dao về chủ đề j Vào 11:52, T.3, 22 Th2, 2022 Nga Lê ***@***.***> đã viết:
> […](#)
> Bình tễnh để t kiếm mấy câu ca dao cho Vào 11:28, T.3, 22 Th2, 2022 Phạm Ngọc Hương Thảo < ***@***.***> đã viết: > chịu luông :)) @baonhu1304 <https://github.com/baonhu1304> @letu25082006 > <https://github.com/letu25082006> @NguyenNgocNhuY > <https://github.com/NguyenNgocNhuY> @Lenga10A2 > <https://github.com/Lenga10A2> > > — > Reply to this email directly, view it on GitHub > <[#31 (comment)](https://github.com/baonhu1304/Lop-10A2-nhom-1/discussions/31#discussioncomment-2225726)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/AV6DO6QLKPW4OFEIPCQIRFLU4MGHLANCNFSM5O4S2JKA> > . > You are receiving this because you were mentioned.Message ID: > ***@***.*** > com> >

con ko đọc kĩ đề rồi, thầy cũng đã giải thích rồi, chủ đề tự do, miễn nó làm con thấy nó phù hợp với suy nghĩ hiện tại
